webapp/src/main/java/life/qbic/usermanagement/registration/EmailConfirmationLinkSupplier.java

```diff
@@ -0,0 +1,25 @@
+package life.qbic.usermanagement.registration;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+/**
+ * Supplies email confirmation routes
+ *
+ * @since 1.0.0
+ */
+@Service
+public class EmailConfirmationLinkSupplier {
+
+  @Value("${server.address}")
+  private String host;
+  @Value("${server.port}")
+  private String port;
+  @Value("${email-confirmation-endpoint}")
+  private String emailConfirmationRoute;
```

Comment:
Why the terminology switch between route and endpoint? You also have that in the properties file where you use both terms as well

Reply:
Good point I will switch to endpoint everywhere :+1: